openfecwebapp/templates/partials/filters/unique-receipts.html

```diff
@@ -4,13 +4,15 @@
     <ul>
       <li>
         <input id="is-individual" name="is_individual" type="checkbox" value="true" class="tooltip-trigger" aria-describedby="unique-tooltip" checked>
-        <label for="is-individual" class="t-inline-block">Unique only</label>
-        <div class="tooltip__container">
-          <button class="tooltip__trigger" type="button"><span class="u-visually-hidden">Learn more</span></button>
-          <div id="unique-tooltip" role="tooltip" class="tooltip tooltip--under tooltip--left">
-            <p class="tooltip__content">Some transactions — such as earmarked contributions and partnership contributions — are reported more than once for clarity on the printed page but can lead to confusion when viewed as data online. Use the unique only filter to remove these extra transactions.</p>
+        <label for="is-individual" class="t-inline-block">
+          Unique only
+          <div class="tooltip__container">
```

Comment:
I think I wrote the original issue wrong. This should be next to the `legend` above, in order to mimc the transaction period filter, like: 
![image](https://cloud.githubusercontent.com/assets/1696495/17605454/5eaa6804-5fcf-11e6-971f-837bc3e23cf3.png)


Right now, this is including the text of the tooltip in the generated tag: 
![image](https://cloud.githubusercontent.com/assets/1696495/17605447/5685a27e-5fcf-11e6-83bd-d455258c837b.png)